src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -49,12 +57,26 @@ public boolean dataTypeIsNotModifiable(final String typeName) {
 
   @Override
   public String getDateLiteral(final String isoDate) {
-    String literal = super.getDateLiteral(isoDate);
+    String literal;
+
+    // Construct the literal based on whether it is a DATE or TIMESTAMP
     if (isDateTime(isoDate)) {
-      literal = "TIMESTAMP " + literal.replace(' ', 'T');
+      Date date;
+      try {
+        date = new ISODateFormat().parse(isoDate);
+        Instant instant = date.toInstant();
+        OffsetDateTime utcDateTime = instant.atOffset(ZoneOffset.UTC);
+        String formattedDate = utcDateTime.format(ISO_LOCAL_DATE);
+        String formattedTime = utcDateTime.format(ISO_LOCAL_TIME);
+        literal = "TIMESTAMP '" + formattedDate + "T" + formattedTime + "Z'";
```

Comment:
nit: just return directly here instead of assigning to `literal`

Reply:
done